plugins/optimization-detective/load.php

```diff
@@ -25,6 +25,21 @@
 	return;
 }
 
+if ( ! file_exists( __DIR__ . '/detection/web-vitals.asset.php' ) ) {
```

Comment:
@westonruter Can't we wrap it in `is_admin()`? Users will face this problem while activating the plugin. So having this in admin should suffice? In production this check will be redundant anyway.

Reply:
Good idea! Did that in ae07548a as well as checking if WP-CLI.

Reply:
I think this is good now as otherwise it would need to `require` the PHP file on every request when it may not be needed.